modules/services/window-managers/i3-sway/sway.nix

```diff
@@ -394,12 +409,12 @@ in {
     home.packages = optional (cfg.package != null) cfg.package
       ++ optional cfg.xwayland pkgs.xwayland;
     xdg.configFile."sway/config" = {
-      source = configFile;
+      source = checkSwayConfig;
       onChange = ''
         swaySocket=''${XDG_RUNTIME_DIR:-/run/user/$UID}/sway-ipc.$UID.$(${pkgs.procps}/bin/pgrep -x sway || ${pkgs.coreutils}/bin/true).sock
         if [ -S $swaySocket ]; then
           echo "Reloading sway"
-          $DRY_RUN_CMD ${pkgs.sway}/bin/swaymsg -s $swaySocket reload
+          $DRY_RUN_CMD ${cfg.package}/bin/swaymsg -s $swaySocket reload
```

Comment:
Is there a reason this has been changed (apart from it maybe being an oversight in the original code)?

Reply:
No reason beyond what you said. This makes i3 and Sway configuration similar in that regard